app/components/filters/filter.ts

```diff
@@ -21,8 +22,8 @@ export default class FilterComponent<S = Signature> extends Component<S> {
    * @param param name of the queryParameter that is presented to the user
    * @returns queryParameter value. Possibly undefined
    */
-  getQueryParam(param: string) {
-    return get(this.router.currentRoute.queryParams, param);
+  getQueryParam(param: string): string | undefined {
```

Comment:
**Question:** Any reason to add the type here? Is the type inference broken?

Reply:
No, just a personal preference for explicit return types, but I can remove it.

Reply:
I found a post that describes pretty well why I prefer explicit return types: https://stackoverflow.com/a/70001850